_includes/releases/v23.1/v23.1.0-alpha.3.md

```diff
@@ -0,0 +1,103 @@
+- The [load-based splitter](https://www.cockroachlabs.com/docs/v22.2/load-based-splitting) now supports using request CPU usage to split ranges. This is introduced with the previous cluster setting `kv.allocator.load_based_rebalancing.objective`, which when set to `cpu`, will use request CPU usage. The threshold above which CPU usage of a range is considered for splitting is defined in the cluster setting `kv.range_split.load_cpu_threshold`, which has a default value of `250ms`. [#96128][#96128]
+- Added the flag `--disable-max-offset-check` to disable node self-termination when it detects [clock skew](https://www.cockroachlabs.com/docs/stable/operational-faqs.html#what-happens-when-node-clocks-are-not-properly-synchronized) with the rest of the cluster beyond `--max-offset`. The operator assumes responsibility for ensuring that real clock skew never exceeds `--max-offset`. [#96141][#96141]
+
+<h3 id="v23-1-0-alpha-3-db-console-changes">DB Console changes</h3>
+
+- Added execution insights to the [Statement Fingerprint Details](https://www.cockroachlabs.com/docs/cockroachcloud/statements-page.html#sql-statement-fingerprints) and [Transaction Fingerprint Details](https://www.cockroachlabs.com/docs/stable/ui-transactions-page.html#transaction-fingerprints-view) Pages. [#96440][#96440]
+- Txn insights that were not found will now display a message `Insight not found`. [#96832][#96832]
+- Added waiting statement ID and fingerprint to the [insights transaction](https://www.cockroachlabs.com/docs/stable/ui-insights-page.html#transaction-executions-view) details page. Added blocking transaction ID and fingerprint to the the [insights statement](https://www.cockroachlabs.com/docs/stable/ui-insights-page.html#statement-executions-view) page. [#96872][#96872]
+
+<h3 id="v23-1-0-alpha-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where casting a [TimeTZ](https://www.cockroachlabs.com/docs/stable/time.html) to an array results in displaying second offsets even if they are zero. [#96583][#96583]
+- Allowed [`ALTER TABLE .. ADD/DROP CONSTRAINT .. NOT VALID`](https://www.cockroachlabs.com/docs/stable/alter-table.html) and [`VALIDATE CONSTRAINT ..`](https://www.cockroachlabs.com/docs/stable/alter-table.html#validate-constraints) to behave consistently with PostgreSQL. Previously, the `VALIDATE CONSTRAINT` would fail and cause the whole statement to fail.[#96648][#96648]
+- Resolved the [TimestampTZ](https://www.cockroachlabs.com/docs/stable/timestamp.html) to match PostgreSQL. We previously included the minute/second offset for TimestampTZ in certain places when casting it to string even when they were zero. [#96833][#96833]
```

Comment:
`TIMESTAMPTZ`

"Previously, the minute/second offset was included for `TIMESTAMPTZ` in certain places when casting it to a [`STRING`](string.html), even when they were zero."